Committed for glibc 2.33

commit cd41ffeb0bbca641a02e1030e13ab317ce7d645c
Author: Samuel Thibault <samuel.thibault@ens-lyon.org>
Date:   Thu Aug 27 13:35:58 2020 +0200

    hurd: define BSD 4.3 ioctls only under __USE_MISC

diff --git a/sysdeps/mach/hurd/bits/ioctls.h b/sysdeps/mach/hurd/bits/ioctls.h
index eed63e759b..1226186ff9 100644
--- a/sysdeps/mach/hurd/bits/ioctls.h
+++ b/sysdeps/mach/hurd/bits/ioctls.h
@@ -321,6 +321,7 @@ enum __ioctl_datum { IOC_8, IOC_16, IOC_32, IOC_64 };
 /* Compatibility with 4.3 BSD terminal driver.
    From 4.4 <sys/ioctl_compat.h>.  */
 
+#ifdef __USE_MISC
 #ifdef USE_OLD_TTY
 # undef  TIOCGETD
 # define TIOCGETD	_IOR('t', 0, int)	/* get line discipline */
@@ -428,5 +429,6 @@ enum __ioctl_datum { IOC_8, IOC_16, IOC_32, IOC_64 };
 # define EXTA	14
 # define EXTB	15
 #endif /* USE_OLD_TTY */
+#endif
 
 #endif /* bits/ioctls.h */
